package.json

```diff
@@ -168,27 +168,12 @@
   "devDependencies": {
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
+    "@netlify/eslint-config-node": "^0.2.6",
     "@oclif/dev-cli": "^1.22.2",
     "@oclif/test": "^1.2.5",
     "auto-changelog": "^2.0.0",
     "ava": "^2.4.0",
-    "babel-eslint": "^10.1.0",
     "cross-env": "^7.0.2",
-    "eslint": "^7.11.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-ava": "^11.0.0",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-html": "^6.1.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-markdown": "^1.0.2",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.20.5",
-    "eslint-plugin-standard": "^4.0.1",
-    "eslint-plugin-unicorn": "^23.0.0",
-    "eslint-plugin-you-dont-need-lodash-underscore": "^6.10.0",
```

Comment:
Those are all `dependencies` of `@netlify/eslint-config-node`.

Technically, the "proper" way is to define those as `peerDependencies` of `@netlify/eslint-config-node`, then install those as `devDependencies` of each project that uses `@netlify/eslint-config-node`. However, this would mean each time one ESLint plugin has a new version, we would need to merge Renovate PRs in each project.

Installing those as `dependencies` of `@netlify/eslint-config-node` instead works due to a hack: npm/yarn dependencies deduping makes those available to the top-level `node_modules/*` directory instead of `node_modues/@netlify/eslint-config-node/node_modules/*`. Without this deduping, this would not work because ESLint `require()` those plugins directly in the parent project's root directory.

npm@7 might solve this since `peerDependencies` are now automatically installed by parent projects, which might solve this problem.